Examples/Detectors/TGeoDetector/src/TGeoDetector.cpp

```diff
@@ -437,6 +437,8 @@ void TGeoDetector::addOptions(
   // required global options
   opt("geo-tgeo-filename", value<std::string>()->default_value(""),
       "Root file name.");
+  opt("geo-tgeo-worldvolume", value<std::string>()->default_value(""),
```

Comment:
Is this key used anywhere?

Reply:
I could not find any place it is used. It was just a test, if this missing option is responsible for Pavels problem. I will close this now